js/okx.js

```diff
@@ -1976,14 +1976,14 @@ module.exports = class okx extends Exchange {
         /**
          * @method
          * @name okx#createOrder
-         * @description create a trade order
+         * @description create a trade order, *reduceOnly must be specified for swap/future to reduce a position*
```

Comment:
These changes are completely unrelated, please revert them

Reply:
@samgermain also don't forget to revert this

Reply:
In the future, if it's a simple commit like this that you don't think should be removed, could you just merge it anyway. When important commits like this that other commits are built on don't get merged, it really slows down progress

Reply:
@samgermain I'm not sure I fully understand your point but I'm insisting on getting all unrelated changes removed because if something goes wrong and we need to revert it we can pick up the merge commit and revert the entire PR really quickly, if you have here unrelated unchanged those would be reverted as well unnecessarily